tests/README.md

```diff
@@ -50,6 +50,10 @@ To run the same tests on a release build, replace `debug` with `release`.
 # Functional tests
 
 [TODO]
+Functional tests are located under the `tests/functional` directory. 
+Requires installing: https://github.com/emesik/monero-python
+Make sure you run the monero daemon with rpc port 28081, then start tests by invoking ./blockchain.py or ./speed.py
```

Comment:
Is this right ? This introduces a whole new network type with a new default port. Presumably these tests use it ?

Reply:
My mistake. If we keep the regtest network type, it should indeed be adjusted to 48081 mentioned in the cryptonote_config.h. If we do not create a new network type, port 28081 is fine as tests can occur in testnet.